src/commands/dora/__tests__/deployment.test.ts

```diff
@@ -0,0 +1,161 @@
+// import {default as axios} from 'axios'
+import {Cli} from 'clipanion/lib/advanced'
+
+import {createMockContext} from '../../../helpers/__tests__/fixtures'
+import * as gitFunctions from '../../../helpers/git/get-git-data'
+
+import {SendDeploymentEvent} from '../deployment'
+import {DeploymentEvent} from '../interfaces'
+
+describe('deployment', () => {
+  describe('getApiHelper', () => {
+    test('should throw an error if API key is undefined', () => {
+      process.env = {}
+      const write = jest.fn()
+      const command = new SendDeploymentEvent()
+      command.context = {stdout: {write}} as any
+
+      expect(command['getApiHelper'].bind(command)).toThrow('API key is missing')
+      expect(write.mock.calls[0][0]).toContain('DD_API_KEY')
+    })
+  })
+})
+
+const makeCli = () => {
+  const cli = new Cli()
+  cli.register(SendDeploymentEvent)
+
+  return cli
+}
+
+describe('execute', () => {
+  const runCLI = async (extraArgs: string[], extraEnv?: Record<string, string>) => {
+    const cli = makeCli()
+    const context = createMockContext() as any
+    process.env = {DD_API_KEY: 'PLACEHOLDER', ...extraEnv}
+    context.env = process.env
+    const code = await cli.run(['dora', 'deployment', ...extraArgs], context)
+
+    return {context, code}
+  }
+  describe('dry-run', () => {
+    const fakeCurrentDate = new Date(1699960651)
+    beforeAll(() => {
+      jest.useFakeTimers()
+      jest.setSystemTime(fakeCurrentDate)
+    })
+    afterAll(() => {
+      jest.useRealTimers()
+    })
+    test('with all parameters provided', async () => {
+      /* eslint-disable prettier/prettier */
+      const {context, code} = await runCLI([
+        '--dry-run',
+        '--service', 'test-service',
+        '--started-at', '1699960648',
+        '--finished-at', '1699960650',
+        '--env', 'test',
+        '--git-repository-url', 'https://github.com/DataDog/datadog-ci',
+        '--git-commit-sha', '2186e1b0ab2a87e312a8c831d5bc947fa081d4f9',
+      ])
+      /* eslint-enable prettier/prettier */
+      expect(code).toBe(0)
+      checkDryRunConsoleOutput(context.stdout, {
+        service: 'test-service',
+        startedAt: new Date(1699960648000),
+        finishedAt: new Date(1699960650000),
+        env: 'test',
+        git: {
+          repoURL: 'https://github.com/DataDog/datadog-ci',
+          commitSHA: '2186e1b0ab2a87e312a8c831d5bc947fa081d4f9',
+        },
+      })
+    })
+    test('with minimal parameters provided', async () => {
+      /* eslint-disable prettier/prettier */
+      const {context, code} = await runCLI([
+        '--dry-run', '--skip-git',
```

Comment:
would this be more consistent? 
```suggestion
        '--dry-run',
        '--skip-git',
```